src/com/magento/idea/magento2plugin/actions/generation/dialog/NewModuleDialog.java

```diff
@@ -245,8 +246,28 @@ public String getPackageName() {
         return this.packageName.getText().trim();
     }
 
+    /**
+     * Getter for Module Name.
+     *
+     * @return String
+     */
     public String getModuleName() {
-        return this.moduleName.getText().trim();
+        return this.removeSubstringFormString(
```

Comment:
if I want to create a module with name **TestModuleAdjustments** I will get **TestAdjustments**.
It is not expected behaviour. Please, remove the **removeSubstringFormString** method. You should revert your changes from that method and the **removeSubstringFormString** method declaration at all.

Reply:
Also, we do not need to add **this** to class methods and variable names if there are no naming conflicts in the current scope.